File: easy/math/Alternating Digit Sum.cpp
class Solution {
public:
    int alternateDigitSum(int n) {
        string s = to_string(n);
        int ans = 0, c = 1;
        for (char i : s) {
            ans += c * (i - '0');
            c *= -1;
        }
        return ans;
    }
};

File: easy/math/Alternating Digit Sum.java
class Solution {
    public int alternateDigitSum(int n) {
        String s = Integer.toString(n);
        int ans = 0, c = 1;
        for (int i = 0; i < s.length(); i++) {
            int x = s.charAt(i) - '0';
            ans += c * x;
            c *= -1;
        }
        return ans;
    }
}
